game.py

```diff
@@ -284,9 +287,9 @@ def _game_over(self):
         for x in range(2, 6):
             self._gameover.append(
                 Sprite(self._sprites,
-                       xoffset + (x - 0.5) * (self._dot_size + 50),
-                       y * (self._dot_size + 200 + self._space) + yoffset,
-                       self._new_dot_gameover('#A0FFA0')))
+                       xoffset + (x - 0.5) * (self._dot_size + 140),
+                       y * (self._dot_size + 140 + self._space) + yoffset,
+                       self._new_dot_gameover(self._colors[FILL])))
```

Comment:
Might be better if these values are stored as a constant as they're reused later.